packages/tempus/src/Tempus_IntegratorBasic_impl.hpp

```diff
@@ -195,7 +195,8 @@ setInitialState(Scalar t0,
     x0->clone_v(), xdot, xdotdot, stepper_->getDefaultStepperState()));
 
   // Set SolutionState MetaData from TimeStepControl
-  newState->setTime    (timeStepControl_->getInitTime());
+  //newState->setTime    (timeStepControl_->getInitTime());
```

Comment:
Why is this no longer needed?

Reply:
We should be using t0 coming into the function.